validator/app_versions.json

```diff
@@ -135,7 +138,10 @@
          "45.0a1", "45.0a2", "45.0", "45.*",
          "46.0a1", "46.0a2", "46.0", "46.*",
          "47.0a1", "47.0a2", "47.0", "47.*",
-         "48.0a1", "48.0a2", "48.0", "48.*"
+         "48.0a1", "48.0a2", "48.0", "48.*",
+         "49.0a1", "49.0a2", "49.0", "49.*",
+         "50.0a1", "50.0a2", "50.0", "50.*",
+         "51.0a1", "51.0a2", "51.0", "51.*"
```

Comment:
Not so easily, we're resolving whatever the user enters as
maxversion/minversion for these apps, so I thought adding a few to not
run into unnecessary errors might be a good idea


On Wed, Sep 7, 2016, at 11:43 AM, Andreas Wagner wrote:
> In validator/app_versions.json[1]:


> > @@ -135,7 +138,10 @@ >           "45.0a1", "45.0a2", "45.0", "45.*",
> > >           "46.0a1", "46.0a2", "46.0", "46.*", >
> > "47.0a1", "47.0a2", "47.0", "47.*", > -         "48.0a1", "48.0a2",
> > "48.0", "48.*" > +         "48.0a1", "48.0a2", "48.0", "48.*", > +
> > "49.0a1", "49.0a2", "49.0", "49.*", > +         "50.0a1", "50.0a2",
> > "50.0", "50.*", > +         "51.0a1", "51.0a2", "51.0", "51.*"
>
> I don't think there will be any release versions of Thunderbird until
> 52. Also, I'm fairly certain we can remove support for all
> applications except Firefox Desktop since we haven't had compat tests
> for any of these in years.
> — You are receiving this because you authored the thread. Reply to
> this email directly, view it on GitHub[2], or mute the thread[3].
>

Links:

  1. https://github.com/mozilla/amo-validator/pull/466#discussion_r77791097
  2. https://github.com/mozilla/amo-validator/pull/466/files/9a22865ee6c3ba4a2fff61fa57e358f8005e4cab#r77791097
  3. https://github.com/notifications/unsubscribe-auth/AAIfGQagnTFdQ8U-cOCSH5OWm3IubJxYks5qnocvgaJpZM4J2s5A